lib/fog/google/requests/storage_json/put_object_acl.rb

```diff
@@ -0,0 +1,49 @@
+module Fog
+  module Google
+    class StorageJSON
+      class Real
+        # TODO: move this methods to helper to use them with put_bucket_acl request
+        def tag(name, value)
```

Comment:
Unused method argument - `name`. If it's necessary, use `_` or `_name` as an argument name to indicate that it won't be used. You can also write as `tag(*)` if you want the method to accept any arguments but don't care about them.<br>Unused method argument - `value`. If it's necessary, use `_` or `_value` as an argument name to indicate that it won't be used. You can also write as `tag(*)` if you want the method to accept any arguments but don't care about them.